.github/workflows/ci.yml

```diff
@@ -8,17 +8,19 @@ on:
 
 env:
   BUILDER_VERSION: v0.6.0
+  BUILDER_SOURCE: releases
   BUILDER_HOST: https://d19elf31gohf1l.cloudfront.net
   PACKAGE_NAME: aws-iot-device-sdk-cpp-v2
   LINUX_BASE_IMAGE: ubuntu-16-x64
-  RUN: ${{ github.run_id }}-${{ github.run_number }}
```

Comment:
If you remove this, you will get cached results from cloudfront, which can be a problem.